### PR TITLE
Edit action for time entries

### PR DESCRIPTION
**Pull request by mathiazom**
Added action for editing existing time entries, with shortcut `cmd`+`e`

### CODE REVIEW

File: src/Forms/SubmitHours.tsx [hunk 1]
```diff
@@ -17,26 +17,35 @@ type FormValues = {
...
   initialDate?: Date;
+  notes?: string | null;
   skipToSubmit?: boolean;
 }) {
   const { push } = useNavigation();
   const { handleSubmit, itemProps, setValidationError, values } = useForm<FormValues>({
     initialValues: {
       spent_date: initialDate,
       hours: hours,
+      notes: notes ?? undefined
```
Comment (adriansberg) [inline src/Forms/SubmitHours.tsx:39]: Maybe the `notes` in `FormValues` can be updated to be of type `string | null`, then we can just pass `notes` as is.

**Review by adriansberg — APPROVED** [review]
Works like a charm :shipit:

File: src/Forms/SubmitHours.tsx (hunk 1 above)
Comment (mathiazom) [reply src/Forms/SubmitHours.tsx:39]: Good point :+1: Only issue is that the `value` prop in `Form.TextArea` only accepts `string | undefined` 🙃

Comment (adriansberg) [reply src/Forms/SubmitHours.tsx:39]: Ah, nevermind me then 👍

Comment (mathiazom) [reply src/Forms/SubmitHours.tsx:39]: So you might have to do something like this instead

image

Comment (adriansberg) [reply src/Forms/SubmitHours.tsx:39]: 🤮

**Pull request merged**